server/src/test/java/com/cloud/resource/ResourceManagerImplTest.java

```diff
@@ -169,41 +167,6 @@ public void setup() throws Exception {
         when(configurationDao.getValue(ResourceManager.KvmSshToAgentEnabled.key())).thenReturn("true");
     }
 
-    @Test
-    public void testCheckAndMaintainEnterMaintenanceMode() throws NoTransitionException {
```

Comment:
@anuragaw Can you add unit tests for the new cases, or instead of removing old ones fix/refactor them?

Reply:
Doing it @rhtyd